packages/asc-ui/src/components/Alert/Alert.stories.tsx

```diff
@@ -82,7 +92,7 @@ export const NormalState = () => (
 
 export const WithCustomChildren = () => (
   <>
-    <Alert level="normal" dismissible>
```

Comment:
The error alert has been removed here, is that intentional?

Reply:
I just changed `<Alert level="normal" dismissible>` to `<Alert level="warning" dismissible>`